book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -2,7 +2,7 @@
 
 This is a list of production Rust code that is in scope and out of scope for Zebra's first audit.
 
-Test code, deployment configurations, and other configuration files in the `zebra` repository are out of scope.
+Test code, deployment configurations, and other configuration files in the `zebra` repository are out of scope. Due to the way we've created the `audit-v1.0.0-rc.0` branch, they might not compile, run, or pass.
```

Comment:
Nitpick:

```suggestion
Test code, deployment configurations, and other configuration files in the `zebra` repository are out of scope. Due to the way we've created the `audit-v1.0.0-rc.0` branch, tests might not compile, run, or pass.
```

If it would be useful to auditors and it's easy to fix, we may want to update the branch so they'll compile.

Reply:
> If it would be useful to auditors and it's easy to fix, we may want to update the branch so they'll compile.

The priority for this branch was significant changes in auditable code.

So to avoid merge conflicts in the tests, I didn't apply any test changes or fixes, unless they were part of a significant change PR. So even if we get the tests compiling, I'm not sure if they will pass.

We also agreed to limit the time we spent on the audit branch to 1 day, so we only have a few hours left to make any changes.